third-party/tidb-operator/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@kubernetes-models/tidb-operator",
+  "version": "0.1.0",
```

Comment:
Changeset will bump this field automatically.

```suggestion
  "version": "0.0.0",
```

Reply:
Fixed, thanks!